coralproject/talk
# DSA final copy updates

## What does this PR do?

- Final copy text for titles and descriptions on notifications
- Some new icons for notifications (rejected comment icon)
- Updated styles for the embedded comment on some notifications, updates the text and makes the toggle-able comment component more re-usable

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- create some comments
- DSA report and decide on the report for some of the comments
- regularly reject some comments in the mod queues
- check the notifications for both the reporters and the commenters to see new copy and styles in notifications area

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into DSA epic branch
